config/app.default.php

```diff
@@ -92,7 +92,7 @@
         // 'cacheTime' => '+1 year'
```

Comment:
Why is this not enabled by default (and in app_local set to +1 second while in app.php set to +1 year?
It is common to long-cache assets via http headers of the web server anyway isnt it?

Reply:
Unrelated to this PR.